### PR TITLE
all necessary logic for calling VotiumBribeProcessor

### PR DESCRIPTION
**Pull request by jorijnsmit**
added a manual override prompt in case the `OnChainPricing` returns a better quote than cowswap:
```
cowswap quotes: 1000.0 ALCX for 22.5098948521604 WETH
sushi quotes:   1000.0 ALCX for 22.52643031583673 WETH
pass sushi's quote to cowswap instead? [y/n]:
```

### CODE REVIEW

File: scripts/badger/process_bribes.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+def step2():
+    badger_share = int(WETH.balanceOf(SAFE) * BADGER_SHARE)
+    cvx_share = WETH.balanceOf(SAFE) - badger_share
+    assert badger_share + cvx_share == WETH.balanceOf(SAFE)
+
+    order_payload, order_uid = SAFE.cow._sell(
+        asset_sell=WETH, mantissa_sell=badger_share, asset_buy=BADGER, coef=.99
+    )
+    SAFE.swapWethForBadger(order_payload, order_uid)
```
Comment (petrovska-petro) [inline scripts/badger/process_bribes.py:108]: should not this be `PROCESSOR` instead?

similar for L113, L117, L118

missing in this method at the end `SAFE.post_safe_tx()`

File: scripts/badger/process_bribes.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            mantissa_sell=balance,
+            buy_token=WETH,
+            coef=.99,
+            prod=COW_PROD
+        )
+        PROCESSOR.sellBribeForWeth(order_payload, order_uid)
+
+
+def step2():
+    badger_share = int(WETH.balanceOf(SAFE) * BADGER_SHARE)
```
Comment (petrovska-petro) [inline scripts/badger/process_bribes.py:101]: here i think also requires to check instead the `PROCESSOR` balance after `sellBribeForWeth` is exec

File: scripts/badger/process_bribes.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    if sim:
+        from brownie_tokens import MintableForkToken
+        alcx = MintableForkToken(registry.eth.treasury_tokens.ALCX)
+        alcx._mint_for_testing(SAFE.badger.strat_bvecvx, 500e18)
+        claimed = {alcx.address: 500e18}
+    else:
+        claimed = SAFE.badger.claim_bribes_votium(
+            registry.eth.bribe_tokens_claimable
+        )
+    for addr, mantissa in claimed:
```
Comment (petrovska-petro) [inline scripts/badger/process_bribes.py:42]: i think intention here is `claimed.items()`

**Comment by jorijnsmit** [comment]
thanks @petrovska-petro, fixed now.

**Review by petrovska-petro — APPROVED** [review]
LGTM, lets see it in action 🔥

**Comment by GalloDaSballo** [comment]
Step wise logic LGTM

**Review by GalloDaSballo — APPROVED** [review]
LGTM, see above for note on order pricing

**Pull request merged**